modules/base/templates/backups/miraheze-backup.py.erb

```diff
@@ -14,10 +14,12 @@ from fabric import Connection
 from datetime import datetime
 
 parser = argparse.ArgumentParser(description='Provides backup and download functionality for Miraheze backups from PCA.')
-parser.add_argument('action', choices=['backup', 'download', 'find', 'unfreeze'], help='Action to be ran')
+parser.add_argument('action', choices=['backup', 'download', 'find', 'delete', 'unfreeze'], help='Action to be ran')
 parser.add_argument('type', choices=['private', 'sslkeys', 'phabricator', 'sql', 'mediawiki-xml'], help='Type of backup to handle using the action')
 parser.add_argument('--date', dest='date', help='Date for backup to download', metavar='YYYY-MM-DD')
 parser.add_argument('--database', dest='database', help='Specific database to download or backup')
+parser.add_argument('--delete-all', dest='delete_all', required=False, action='store_true', default=False, help='Delete all backups for the "type" selected')
```

Comment:
This would also delete every backup - meaning you lose all historic archives that you might want to intentionally keep - therefore this feature seems really dangerous to me if you don't restrict it by date.